src/root_files/_common/404.html

```diff
@@ -21,8 +20,7 @@
         'pl': 'po',
         'pt': 'br|mz|ao|pt|gw|pg|cv|st',
         'ru': 'ru|ua|by|kz',
-        // TODO: uncomment if we enable th again
-        // 'th': 'th',
```

Comment:
German and Thai languages are supported for a long time already, so it looks pretty much like these TODO's must have been forgotten about.